src/gateways/createHighlightClient.ts

```diff
@@ -8,19 +9,23 @@ const formatError = (response: Response) => {
     });
 };
 
-export default (url: string) => {
+export default (
+  basePath: string,
+  getAuthorizedFetchConfig: () => Promise<FetchConfig> = () => Promise.resolve({credentials: 'include'})
```

Comment:
does this default value ever get used?

Reply:
In tests, I think. Maybe prerendering. I could probably just update them all though.

Reply:
All those places are fine with the default being `() => Promise.resolve({})`. I think it's a fine default for prerendering etc but let me know if you want it taken out.